js/jquery.dvfTables.js

```diff
@@ -64,6 +65,11 @@
 
       this.config.columns = columns;
 
+      return this;
+    },
+
+    getTableOptions: function () {
+      this.config = $.extend(this.config, this.options.tableOptions);
```

Comment:
After all my suggestions `this.options.tableOptions` may not be set so we need to check before using it.

```
parseTableOptions: function () {
  if (this.options.tableOptions) {
    this.config = $.extend(this.config, this.options.tableOptions);
  }

  return this;
}
```